cmd/root.go

```diff
@@ -39,7 +39,7 @@ func NewRootCMD() *cobra.Command {
 
 func setFlags(cmd *cobra.Command, v *viper.Viper) {
 	flags := cmd.Flags()
-	flags.String("alternate-language", "", "(deprecated)")
+	flags.String("alternate-language", "", "Optional alternate language name. Auto-discovered language takes priority.")
```

Comment:
What about?
```
Optional language name. Auto-detected language takes priority.
```

Reply:
Yeah this sounds better. Will update along with `alternate-project` info, to keep it consistent :+1: